lib/posix-futex/futex.c

```diff
@@ -382,14 +382,25 @@ UK_LLSYSCALL_R_DEFINE(pid_t, set_tid_address, pid_t *, tid_ref)
 	return self_tid;
 }
 
-/* Thread exit handler that clears child TID at the stored reference */
-static void pfutex_child_cleartid_term(struct uk_thread *child __unused)
+static void thread_exit_handler(struct uk_thread *child __unused)
```

Comment:
Is the `__unused` attribute still needed now? As far as I can see `child` is used at line 399.

Reply:
Good catch, thanks! 👍🏼